cognite/processpool/processpool.py

```diff
@@ -129,14 +135,14 @@ def __init__(self, worker_class: type, pool_size: int = 1):
 
         self._pool = [self._create_new_worker() for _ in range(pool_size)]
 
-        self._job_queue = queue.Queue()  # type: queue.Queue # no need for a MP queue here
+        self._job_queue: queue.Queue[Optional[Job]] = queue.Queue()  # no need for a MP queue here
```

Comment:
Why does it work here without quotation marks, but above it requires them?

Reply:
Not sure, but this is only queue.Queue, not multiprocessing

Reply:
Had to add the quotation marks anyway for it to work with python 3.8